lib/runtime-core/Cargo.toml

```diff
@@ -18,6 +18,8 @@ errno = "0.2.4"
 libc = "0.2.49"
 hex = "0.3.2"
 smallvec = "0.6.9"
+bincode = "1.1"
+colored = "1.8"
```

Comment:
Should `colored` be moved to a higher level project like `wasmer` 🤔

Reply:
How can we use dependencies in the top level workspace in `wasmer-runtime-core`?

Reply:
I think some user, like runtime and runtime-core users may not have a need for colored output since the usage is not in a terminal, for these cases could this be moved to a higher level project?

Reply:
I think we can change this later when there's a need for high modularity. Currently I think it's okay to simulate Rust libstd's behavior on panic (output to stderr).